tsconfig.json

```diff
@@ -16,15 +16,9 @@
     "jsx": "react-jsx",
     "baseUrl": ".",
     "paths": {
-      "@/components/*": [
-        "components/*"
-      ],
-      "@/utils/*": [
-        "utils/*"
-      ],
-      "@/data/*": [
-        "pages/data/*"
-      ]
```

Comment:
Prettier version issues here 🤔 . Also running the dev server is changing `jsx` to `preserve` rather than `react-jsx`.

Reply:
Yeah i had that too in my latest PR, just discarded the changes to this file. But if we're all having it then it's something we'll have to look into :/

Reply:
I think we actually should add `"react/react-in-jsx-scope": "off"` to the lint config to prevent this warning 💭